src/main/java/frc/robot/RobotContainer.java

```diff
@@ -61,431 +66,464 @@
+        operatorController
+                .povRight()
+                .onTrue(new InstantCommand(() -> preppedLocation = IntakeClimbLocation.RIGHT));
+        /* Height for scoring */
+        operatorController
+                .a()
+                .onTrue(new InstantCommand(() -> preppedHeight = ElevatorHeight.SCORE_L4));
+        operatorController
+                .b()
+                .onTrue(new InstantCommand(() -> preppedHeight = ElevatorHeight.SCORE_L3));
+        operatorController
+                .x()
+                .onTrue(new InstantCommand(() -> preppedHeight = ElevatorHeight.SCORE_L2));
+        operatorController
+                .y()
+                .onTrue(new InstantCommand(() -> preppedHeight = ElevatorHeight.SCORE_L1));
+
+        /* Left and right for scoring */
+        operatorController
+                .leftBumper()
+                .onTrue(new InstantCommand(() -> preppedScoringLocation = ScoringLocation.LEFT));
+        operatorController
+                .rightBumper()
+                .onTrue(new InstantCommand(() -> preppedScoringLocation = ScoringLocation.RIGHT));
+        new Rotation2d();
+        /*
+         * Testing code for climb, arm, and elevator
+         * operatorController
+         * .y()
+         * .whileTrue(new RunCommand(() -> arm.testArmMovementUp()));
+         * operatorController
+         * .y()
+         * .onFalse(new InstantCommand(() -> arm.stopArmMovement()));
+         * operatorController
+         * .a()
+         * .whileTrue(new RunCommand(() -> arm.testArmMovementDown()));
+         * operatorController
+         * .a()
+         * .onFalse(new InstantCommand(() -> arm.stopArmMovement()));
+         * operatorController
+         * .povUp()
+         * .whileTrue(new RunCommand(() -> climb.testClimbMovementUp()));
+         * operatorController
+         * .povUp()
+         * .onFalse(new InstantCommand(() -> climb.stopClimbMovement()));
+         * operatorController
+         * .povDown()
+         * .whileTrue(new RunCommand(() -> climb.testClimbMovementDown()));
+         * operatorController
+         * .povDown()
+         * .onFalse(new InstantCommand(() -> climb.stopClimbMovement()));
+         * operatorController
+         * .povRight()
+         * .whileTrue(new RunCommand(() -> elevator.testElevatorMovementUp()));
+         * operatorController
+         * .povRight()
+         * .onFalse(new InstantCommand(() -> elevator.stopElevatorMovement()));
+         * operatorController
+         * .povLeft()
+         * .whileTrue(new RunCommand(() -> elevator.testElevatorMovementDown()));
+         * operatorController
+         * .povLeft()
+         * .onFalse(new InstantCommand(() -> elevator.stopElevatorMovement()));
+        */
+        operatorController.start().onTrue(new InstantCommand(()->drivetrain.resetRotation(Rotation2d.fromDegrees(matchState.isBlue()?0:180)))); // is this correct?
```

Comment:
I looked at last year's code (see below) and think this is correct.

public void resetYaw() {
    resetYawToAngle(matchState.isBlue() ? 0 : 180);
  }